Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6.12-buster
+
+COPY . /vane/
+WORKDIR "/vane/"
+RUN apt-get install -y gcc
```

Comment:
Move the apt and pip install statements above the code copy to speed up the build process.

Reply:
I agree the apt can go above and hence create the layer earlier, but the pip installation cannot occur before the copy as the requirements.txt is not in the container at this stage... I will refactor to include the gcc install above the copy...